app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @shelter = Shelter.find(params[:shelter_id])
+  end
+
+  def create
+    @shelter = Shelter.find(params[:shelter_id])
+    user_name = User.find_by(name: params[:name])
```

Comment:
Since this is a user object, I think we should change this from 'user_name' to 'user'.